Upstream: Yes
Reason: Without this patch tests fail on shells backslash escapes disabled
Url: https://github.com/h2o/h2o/pull/2331
--- a/t/50mruby.t
+++ b/t/50mruby.t
@@ -498,7 +498,7 @@
 EOT
     my $nc = sub {
         my $path = shift;
-        my $cmd = "echo 'GET $path HTTP/1.1\\r\\nHost: 127.0.0.1\\r\\n\\r' | nc 127.0.0.1 $server->{port}";
+        my $cmd = "echo 'GET $path HTTP/1.1\r\nHost: 127.0.0.1\r\n\r' | nc 127.0.0.1 $server->{port}";
         (undef, my $r) = run_prog($cmd);
         split(/\r\n\r\n/, $r, 2);
     };
